docs/studios/unity/tutorials/quickstart_sample_scene.mdx

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_label: 'Quickstart - Sample Scene and Bot'
+---
+
+# Quickstart - Sample Scene and Bot
+
+**_Time to complete: 2 minutes_**
+
+This quick walkthrough will show you an example scene and bot using the Regression Games Unity SDK. Within a few minutes,
+you will have your first bot running in Unity!
+
+
+## Step 1: Create or open a 3D URP scene
+
+To try out the sample scene, you will need to open or create a project that uses Unity's Universal Render Pipeline (URP).
+Within your Unity Hub, search for a template called **3D (URP)**, download it if you haven't already, and create a new
+project from that template.
+
+![Create a new project from the 3D URP template](./quickstart_images/urp.png)
+
+## Step 2: Import the Regression Games Unity Bots SDK
+
+Our package is available via GitHub [here](https://github.com/Regression-Games/RGUnityBots). Import the repository by 
+clicking **Add package from git URL** in the **Package Manager** window (this window can be opened
+in the **Window > Package Manager** menu) and pasting in the following URL.
+
+```
+https://github.com/Regression-Games/RGUnityBots.git#v0.0.15
+```
+
+- Note that this package utilizes TextMeshPro.  If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
+- Note that this package installs a dependency of Newtonsoft Json (https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) for serializing/deserializing Json message payloads.
+
+![Screenshot of the package imported into the project](./first_bot_tutorial_images/tutorial_1_package.png)
+
+## Step 3: Login (optional) and load the scene
+
+Once the package is imported, a welcome pane will open with the ability to login to Regression Games. Feel free to create
```

Comment:
This paragraph doesn't flow smoothly when reading it.

Hopefully our on staff team of language experts will provide better wording :D

Reply:
Reworded this slightly, but looking forward to any other language edits!